modules/core/shared/src/main/scala/scalacache/Cache.scala

```diff
@@ -3,19 +3,20 @@ package scalacache
 import scala.concurrent.duration.Duration
 import scala.language.higherKinds
 
-trait Cache[V] extends CacheAlg[V] {
+//todo merge with alg?
```

Comment:
Is this something we want to address in this PR? Seems like a good candidate for another PR to me, but happy to hear other opinions.